src/main/java/com/quizapp/questiontag/QuestionTagServiceImpl.java

```diff
@@ -8,7 +8,7 @@
 
 @Service
 @RequiredArgsConstructor
-public class QuestionTagServiceImpl implements QuestionTagService {
+public class   QuestionTagServiceImpl implements QuestionTagService {
```

Comment:
Before you push, you can format (CTRL+ALT+L for intelliJ) to avoid. Or at least you can check the "changes" before you commit. You may undo this extra space before QuestionTagServiceImpl.